prql-compiler/src/test.rs

```diff
@@ -3320,3 +3320,20 @@ fn test_params() {
     "###
     );
 }
+
+#[test]
```

Comment:
// TODO: fix this based on https://github.com/PRQL/prql/pull/1818

```suggestion
#[test]
#[should_panic]
```